eip-2470.md

```diff
@@ -161,9 +206,9 @@ The contract has the address above for every chain on which it is deployed.
 
 ## Rationale
 
-SingletonFactory does not allow sending value on create2, this was done to prevent different results on the created object. 
-SingletonFactory allows user defined salt to facilitate the creation of vanity addresses for other projects. If vanity address is not necessary, salt `bytes(0)` should be used.
-Contracts that are constructed by the SingletonFactory MUST not use `msg.sender` in their constructor, all variables must came through initialization data. This is intentional, as if allowing a callback after creation to aid initialization state would lead to contracts with same address (but different chains) to have the same address but different initial state.
+ERC2470 does not allow sending value on create2, this was done to prevent different results on the created object. 
```

Comment:
This is not true anymore, right?

Reply:
Yes, this changes with this PR.  I'll remove this line.